utils/Unpatching.py

```diff
@@ -294,7 +294,8 @@ def fUnpatchLabel(prob_list, patchSize, patchOverlap, actualSize, iClass=0):
     # If iClass=1, the first half unpatchImg[0] is label of image with artifact, the rest unpatchImg[1] for reference images
 
     dOverlap = np.multiply(patchSize, patchOverlap).astype(int)
-    dNotOverlap = np.round(np.multiply(patchSize, (1 - patchOverlap))).astype(int)
+    # dNotOverlap = np.round(np.multiply(patchSize, (1 - patchOverlap))).astype(int)
+    dNotOverlap = np.subtract(patchSize, dOverlap)

```

Comment:
Add option to config file to choose dealing with overlapping parts

Reply:
Round or subtract

Reply:
If the result of np.multiply(patchSize, patchOverlap)=x.5 and np.multiply(patchSize, (1 - patchOverlap))=y.5 the two values will be round to x and y. Then x + y = patchSize-1. To avoid this situation, the subtract should be used in all cases. No need to choose from them.

Reply:
ok, thanks for the clarification